snapcraft.yaml

```diff
@@ -471,22 +450,17 @@ parts:
       - --datadir=/share
       - --datarootdir=/share
       - --includedir=/include
+      - --enable-gs-ps2write
+      - --enable-pdftops
       - --disable-mutool
       - --disable-pdftocairo
       - --disable-acroread
-      - --with-pdftops-path=/snap/cups/current/usr/bin/pdftops
+      - --with-pdftops-path=$CRAFT_STAGE/usr/bin/pdftops
     build-environment:
       # To find the libraries built in this Snap
-      - LD_LIBRARY_PATH: "${LD_LIBRARY_PATH:+$LD_LIBRARY_PATH:}$CRAFT_STAGE/usr/lib:$CRAFT_STAGE/lib"
-      # We have to force-define the macros for the properties of the
-      # external executables here, as the tests do not work in the Snap
-      # build environment
-      - CFLAGS: "$CFLAGS -DHAVE_GHOSTSCRIPT_PS2WRITE -DHAVE_POPPLER_PDFTOPS_WITH_ORIGPAGESIZES -DHAVE_POPPLER_PDFTOPS_WITH_RESOLUTION"
```

Comment:
Do the tests now work with the change in `LD_LIBRARY_PATH`? Have you checked (for example by the summary which `./configure` produces in the end) whether the build gets done correctly?

Reply:
I added the explanation here as both are same. The change in `LD_LIBRARY_PATH` is needed because now I changed the install prefix for `qpdf` from `/` to `/usr`. `qpdf` wasn't respecting `/` prefix and was still installing files under `/usr/`. Using `/usr` as prefix, `qpdf` now installs its libraries under `/usr/lib/<arch-triplet>/`.